index.html

```diff
@@ -171,9 +171,11 @@ <h2>Contact me</h2>
             <form action="https://formspree.io/f/xwkzweoq" method="post" class="forms" id="form">
                 <fieldset class="form-inputs">
                     <input type="text" id="name" placeholder="Your Name" maxlength="30" required />
+
                     <input type="email" id="email" placeholder="example@mail.com" required>
                     <div class="warn"></div>
                     <textarea name="message" placeholder="write your message here" maxlength="500" required></textarea>
+
                     <button type="submit" class="submit-btn">Get In Touch</button>
```

Comment:
- You missed the id of the `textarea`, which is causing a Null Variable error (you can see it on the browser console). This is causing that the data doesn't get stored. Please review this issue so you can move ahead.